vanilla-framework/vanilla-framework
# Vanilla Docs update

## Done

- Standardised heading sizes for docs pages to h2 (with an off-screen h1 at the top)
- Updated the header nav and side nav to Vanilla styling (1.6.4)
- Added GitHub and Report a bug links to component headers
- Added links to design spec for each component
- Added the missing Aside pattern to `metadata.yaml` under Navigation
- Removed TOC on small screens - will be addressed via #1531 

## QA

- Pull code
- Run `cd docs && ./run`
- Open http://0.0.0.0:8104/
- Check that the Header and Side navs use Vanilla styling (reference [here](https://vanilla-framework.github.io/vanilla-framework/examples/patterns/navigation/default/) and [here](https://vanilla-framework.github.io/vanilla-framework/examples/patterns/navigation/sidebar/))
- Check that there are links for GitHub and Report a bug on the component headers (reference [here](https://github.com/ubuntudesign/vanilla-design/issues/119))
- Check that each pattern has a h2 heading
- Check that each pattern has a link to the relevant design spec (if it exists) at the bottom
- Check that the Aside pattern link exists under the Navigation subheading

## Details

Fixes ubuntudesign/docs.vanillaframework.io#115
Fixes ubuntudesign/docs.vanillaframework.io#116
Fixes ubuntudesign/docs.vanillaframework.io#118
Fixes #1205 


## Screenshots
![screenshot sdfsdf](https://user-images.githubusercontent.com/25733845/35151125-02b49b66-fd15-11e7-95b4-49f36f9a5b04.png)
